airflow/providers/databricks/hooks/databricks.py

```diff
@@ -34,6 +34,8 @@
 from airflow.exceptions import AirflowException
 from airflow.hooks.base import BaseHook
 
+RUN_STATE_FORMAT_STRING = "State: {}. Result: {}. {}"
```

Comment:
It’d be better to use named placeholders here to avoid acciedental formatting errors.

Reply:
Although this variable is really only used once and does not really need to be global in the first place? Why not just use an f-string in `get_run_state_str`?

Reply:
it's also used in the test - I first used f-string, but just to avoid changing in two places

Reply:
I think having to change two places is actually kind of a good thing here, since it avoids the string to be accidentally changed—the test failures would notify us the change might be accidental.

Reply:
Ok, will rework it later

Reply:
Ok, addressed comments